LICENSE-2.0.txt

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2014-2022 Snowplow Analytics Ltd.
+   Copyright 2014-2023 Snowplow Analytics Ltd.
```

Comment:
There have been conversations about this, and we were even told at one point that we don't need to update it at all, unless we're making really major changes to a file.  There's also been talk about using sbt plugins and github actions to do this work for us.

But, in this case it was so easy to change every file that I found it easier to make the change rather than to rationalise where it was needed:

```
for file in ${git ls-files); do sed -i 's/2022 Snowplow/2023 Snowplow/'; done
```